src/app/features/genes/components/gene-similar/gene-similar.component.html

```diff
@@ -32,6 +32,7 @@ <h1 class="h2">Similar Genes</h1>
           [genes]="genes"
           [columns]="tableColumns"
           sortField="hgnc_symbol"
+          [sortOrder]="1"
```

Comment:
gene-table defaults to a descending sort, so specify that this table should default to an ascending sort.

Reply:
looks good!